addon/styles/_frost-numeric-select.scss

```diff
@@ -0,0 +1,99 @@
+$height: 30px;
+$border: 0.3px solid $frost-color-input-border;
+$focus-border: 1px solid $frost-color-blue-1 !global;
+$frost-color-button-hover-bg: rgba($frost-color-blue-1, .1) !global;
+$arrow-focus-color: $frost-color-blue-1 !global;
+$error-text-color: $frost-color-input-error-border;
+$disabled-border: 1px solid $frost-color-input-disabled-border;
+$disabled-bg: $frost-color-input-disabled-bg;
+
+.frost-numeric-select {
+  position: relative;
+  display: block;
+  width: 70px;
+  height: $height;
+  margin-bottom: 15px;
+  outline: none;
+
+  &:focus {
+    border: $focus-border;
+  }
+
+  &:disabled {
+    border: $disabled-border;
+    background-color: $disabled-bg;
+  }
+}
+.frost-numeric-select-button {
+  position: relative;
+  display: block;
+  width: 70px;
+  height: $height;
+  margin-bottom: 15px;
+  outline: none;
+  padding-right: 26px;
+
+  &:focus {
+    border: $focus-border;
+  }
+}
+.error {
```

Comment:
Be careful with generic classes - CSS is a global namespace so this could inadvertently apply to elements you wouldn't have expect to impact.  It's much safer to make this `frost-numeric-select-error`